.changeset/orange-jeans-destroy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/alert-dialog': patch
+'@twilio-paste/core': patch
+---
+
+[AlertDialog]: added missing `date-fns` dependency.
```

Comment:
I think these messages got switched- `date-fns` isn't needed for alert dialog 😆

Reply:
🤦 Good catch. Fixed.